pinot-common/src/main/java/org/apache/pinot/common/metadata/segment/SegmentZKMetadata.java

```diff
@@ -409,12 +415,10 @@ public Duration getTimeGranularity() {
 
   @Deprecated
   public Interval getTimeInterval() {
-    String startTimeString = _simpleFields.get(Segment.START_TIME);
-    if (startTimeString != null) {
-      String endTimeString = _simpleFields.get(Segment.END_TIME);
-      TimeUnit timeUnit = TimeUnit.valueOf(_simpleFields.get(Segment.TIME_UNIT));
-      return new Interval(timeUnit.toMillis(Long.parseLong(startTimeString)),
-          timeUnit.toMillis(Long.parseLong(endTimeString)));
+    long startTimeMs = getStartTimeMs();
```

Comment:
Seems like it is deprecated, and my IDE did not find any callers. If so, may be just remove?

Reply:
Will remove all the deprecated ones in a separate PR so that they are easier to track